lib/TransferControllers/TransferReaders/RangeBasedReader.cs

```diff
@@ -137,7 +137,7 @@ protected override void Dispose(bool disposing)
                 await this.DoFetchAttributesAsync();
             }
 #if EXPECT_INTERNAL_WRAPPEDSTORAGEEXCEPTION
-            catch (Exception ex) when (ex is StorageException || ex.InnerException is StorageException)
+            catch (Exception ex) when (ex is StorageException || (ex is AggregateException && ex.InnerException is StorageException))
```

Comment:
"ex is AggregateException", Can you give a bit more explanation on this change?

Reply:
This is because WappedStorageException is removed from netcore, we don't need it.
And since we add inner exception to some kinds of transfer exception, we need to add the code to avoid them to un-wrapperred.